tools/tck-api/src/test/resources/org/opencypher/tools/tck/Foo.feature

```diff
@@ -67,6 +67,22 @@ Feature: Foo
       """
     Then a SyntaxError should be raised at compile time: UnknownFunction
 
+  Scenario: Fail with code
+    Given an empty graph
+    When executing query:
+      """
+      RETURN foo()
+      """
+    Then an error with GQL code fooCode should be raised
```

Comment:
Perhaps we should quote the code? `Then an error with GQL code 'fooCode' should be raised` and/or `Then an error with GQL code "fooCode" should be raised`.

Reply:
It does not seem to matter. The code do not contain spaces, I think, so they are visually clearly delineated. I was thinking about a colon before the message, like this: `Then an error with GQL code fooCode should be raised with message: bla bla bla foo bla`. This seems optically helpful.